samples/client/petstore/ocaml-fake-petstore/src/apis/pet_api.ml

```diff
@@ -0,0 +1,319 @@
+(*
+ * This file has been generated by the OCamlClientCodegen generator for openapi-generator.
+ *
+ * Generated by: https://openapi-generator.tech
+ *
+ *)
+
+let add_pet ~pet_t =
+    let open Lwt.Infix in
+    let uri = Request.build_uri "/pet" in
+    let headers = Request.default_headers in
+    let body = Request.
+        
+        write_as_json_body     
+    
+    
+    
+    
+    
+    
+                Pet.to_yojson
+    
+    
+    
+ pet_t
+    in
+    Cohttp_lwt_unix.Client.call `POST uri ~headers ~body >>= fun (resp, body) ->
+    Request.handle_unit_response resp
+
+let delete_pet ~pet_id ?api_key () =
+    let open Lwt.Infix in
+    let uri = Request.build_uri "/pet/{petId}" in
+    let headers = Request.default_headers in
+    let headers = Request.maybe_add_header headers "api_key"     
+    
```

Comment:
i'll get this merged first and later maybe come up with a PR to remove these blank lines

curious are you using any linter to format the auto-generated code as part of your development workflow?